Jiachen xu

jiachen-ux opened the pull request:
Some UI for Borrow page. 
Main Activity contains a template for making home page buttons.

jiachen-ux committed:
Update Star.java

Add color "blue" to star shape

jiachen-ux committed:
Add blue in shape

jiachen-ux committed:
Merge branch 'master' into Jiachen-Xu

jiachen-ux committed:
Merge branch 'master' into Jiachen-Xu

jiachen-ux committed:
Update Shape.java

jiachen-ux committed:
Merge branch 'master' into Jiachen-Xu

jiachen-ux committed:
Borrw Function

In the Borrow Function,
finished recieve and view books.
Did not finish book's details and maps

Kay7777 reviewed (changes requested):
MainActivity needs to be added the new content of the master branch.

jiachen-ux committed:
Merge branch 'master' into Jiachen-Xu

jiachen-ux committed:
Merge branch 'master' into Jiachen-Xu

jiachen-ux committed:
trash information

Kay7777 reviewed (approved):
LGTM

jiachen-ux committed:
Update BorrowManu.java

1. Added button attributes
2. BorrowManu Implements now OnclickListerner.
3. Using switch to handle different buttons
